packages/core/src/core.ts

```diff
@@ -479,24 +530,33 @@ export class McpServer {
       this.capabilities.tools = { listChanged: true };
     }
 
-    const { mcpInputSchema, validator } = resolveToolSchema(
+    const { mcpInputSchema, validator } = resolveSchema(
```

Comment:
confusing to me to still call it the mcpInputSchema in the return value, e.g., below we have `outputSchemaResolved.mcpInputSchema`